iglo/iglo/settings.py

```diff
@@ -266,6 +270,7 @@ def env(key, as_bool=False, as_list=False, as_int=False, required=True, default=
 ]
 
 ENABLE_AI_ANALYSE_UPLOAD = env("ENABLE_AI_ANALYSE_UPLOAD", as_bool=True, default=False)
+ENABLE_UPCOMING_GAMES_REMINDER = env("ENABLE_UPCOMING_GAMES_REMINDER", as_bool=True, default=False)
```

Comment:
Let's set default=True, otherwise there is no effect.
(or we need also to update deploy/docker-compose.yml)